console/src/tasks.rs

```diff
@@ -11,12 +11,20 @@ use std::{
 };
 use tui::text::Span;
 
+#[derive(Debug)]
+pub(crate) enum ConnectionState {
+    Pending,
```

Comment:
do we ever actually display the pending state? AFAICT, we await the initial connection before we draw the first screen, so while we are in the pending state while waiting to connect, we don't actually draw anything:
https://github.com/tokio-rs/console/blob/ba60d04a71285f96fd1cbc9b86065c1855792681/console/src/main.rs#L32-L38

not opposed to adding this state for future use, but i don't think it's actually displayed to the user currently